docs/resources/tentacle_certificate.md

```diff
@@ -0,0 +1,47 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "octopusdeploy_tentacle_certificate Resource - terraform-provider-octopusdeploy"
+subcategory: ""
+description: |-
+  Generates a X509Certificate for use with a Octopus Deploy Tentacle.
+---
+
+# octopusdeploy_tentacle_certificate (Resource)
```

Comment:
Not sure where the template for this file is, but it would be worth outlining that this certificate is self-signed and should not be used for other purposes.
Feel free to adjust the below as necessary for phrasing.

```
##Octopus certificates
The X.509 certificates generated are self-signed, 2048-bit private keys and intended for use [only between Octopus Server and Tentacle](https://octopus.com/docs/security/octopus-tentacle-communication#Octopus-Tentaclecommunication-Octopuscertificates) communications. There is an insightful discussion of [why Octopus uses self-signed certificates](https://octopus.com/blog/why-self-signed-certificates) by default.

Instead of generating a new certificate through this resource you can use an existing certificate and simply reference the appropriate thumbprint when registering the target.

### State Persistence
This resource that is generated will be stored in the state file and cannot be retrieved later from the external Octopus Server or Tentacle. 

```